Sources/FluentUI_iOS/Components/Notification/MSFNotification.swift

```diff
@@ -61,11 +79,13 @@ import UIKit
     ///   - anchorView: The view used as the bottom anchor for presentation
     ///   (notification view is always presented up from the anchor). When no anchor view is provided the
     ///   bottom anchor of the container's safe area is used.
+    ///  - layoutGuide: The layout guide used to present the toast inside of when no `anchorView` is provided
     ///   - animated: Indicates whether to use animation during presentation or not.
     ///   - completion: The closure to be called after presentation is completed.
     ///   Can be used to call `hide` with a delay.
     @objc public func show(in view: UIView,
                            from anchorView: UIView? = nil,
+                           with layoutGuide: UILayoutGuide? = nil,
```

Comment:
minor: preference - I've tended away from using argument labels (i.e. `with`). For some small cases it helps make things more natural, but especially in functions like this with a lot of parameters, it tends to remove some clarity in the caller in how the parameter is being used

Reply:
Agreed; I'd like to redo the MSFNotification API entirely one of these days (for lots of reasons, but this is one of them).